### DISCUSSION TITLE
How can i use Proxy in fetch

### DISCUSSION THREAD

**Discussion by shahensaifullah** [nuxt/nuxt#17378] [Questions]
In nuxt 2, i used to use axios which have proxy and I used that like this:
```
  axios: {
    withCredentials: true,
    proxy: true
  },
  proxy: {
    '/api/': {
      target: "http://127.0.0.1:8000/",
      pathRewrite: {
        '^/api/': 'api/'
      }
    }
  },
```
It is easy to use proxy, because when I call API just write "/api/user". but axios make this to "http://127.0.0.1:8000/api/user". I don't need to write "http://127.0.0.1:8000/" again and again.
but in nuxt, where it use [ohmyfetch](https://github.com/unjs/ohmyfetch). but cannot make proxy like that. so how can I use proxy in my nuxt 3??

**Comment by Flanker72** [comment]
Already answered here: https://github.com/nuxt/framework/discussions/4343

To create proxy **only for dev environment** you can use `vite.server.proxy` object within your Nuxt3 config.
